<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Добавить заголовок</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QTabWidget" name="tabWidget">
      <widget class="QWidget" name="lab1">
       <attribute name="title">
        <string>Лабораторная работа №1</string>
       </attribute>
       <layout class="QVBoxLayout" name="verticalLayout_5">
        <item>
         <widget class="QGroupBox" name="part1">
          <property name="title">
           <string>Часть 1</string>
          </property>
          <layout class="QVBoxLayout" name="verticalLayout_2">
           <item>
            <widget class="QPushButton" name="pbTask1">
             <property name="text">
              <string>Задание 1</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="pbTask2">
             <property name="text">
              <string>Задание 2</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="pbTask3">
             <property name="text">
              <string>Задание 3</string>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
        <item>
         <widget class="QGroupBox" name="part2">
          <property name="title">
           <string>Часть 2</string>
          </property>
          <layout class="QVBoxLayout" name="verticalLayout_3">
           <item>
            <widget class="QPushButton" name="pbTask4">
             <property name="text">
              <string>Задание 4</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="pbTask5">
             <property name="text">
              <string>Задание 5</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="pbTask6">
             <property name="text">
              <string>Задание 6</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="pbTask7">
             <property name="text">
              <string>Задание 7</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="pbTask8">
             <property name="text">
              <string>Задание 8</string>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
        <item>
         <widget class="QGroupBox" name="part3">
          <property name="title">
           <string>Часть 3</string>
          </property>
          <layout class="QVBoxLayout" name="verticalLayout_4">
           <item>
            <widget class="QPushButton" name="pbTask9">
             <property name="text">
              <string>Задание 9</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="pbTask10">
             <property name="text">
              <string>Задание 10</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="pbTask11">
             <property name="text">
              <string>Задание 11</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="pbTask12">
             <property name="text">
              <string>Задание 12</string>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="tab_2">
       <attribute name="title">
        <string>Tab 2</string>
       </attribute>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
